tags-config-service-impl/src/main/java/org/hypertrace/tag/config/service/TagConfigConstants.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.tag.config.service;
+
+public class TagConfigConstants {
+
+  private TagConfigConstants() {}
+
+  public static final String TAG_CONFIG = "tag-config";
```

Comment:
I am usually curious about the pattern `MY_CONSTANT="my-constant"`, I don't see what is the value added, I think if at least we could specify what is this constant describing would be more helpful e.g. TAG_CONFIG_KEY or CONFIG_KEY (the TAG_ prefix does not seem needed).

Reply:
Changed the names. Hopefully, makes more sense now.